macros/edr/metadata_collection/get_columns_from_information_schema.sql

```diff
@@ -15,13 +15,11 @@
         upper(column_name) as column_name,
         data_type
     from {{ schema_relation.information_schema('COLUMNS') }}
-    where upper(table_schema) = upper('{{ schema_name }}')
+    {%- if schema_name %} where upper(table_schema) = upper('{{ schema_name }}') {%- endif -%}
```

Comment:
This means that if you don't get a schema you select all schemas. Is that intended?

Reply:
yep
but I will remove the condition since I found out yesterday that Bigquery requires a schema name (or else, an error)